specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json

```diff
@@ -3500,54 +3532,72 @@
           }
         ],
         "responses": {
-          "200": {
-            "description": "Successfully deleted hybrid connection."
-          },
-          "404": {
-            "description": "Hybrid connection does not exist."
+          "204": {
+            "description": "No Content"
           }
         }
-      },
-      "patch": {
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/host/default/properties/config": {
+      "get": {
         "tags": [
           "WebApps"
         ],
-        "summary": "Creates a new Hybrid Connection using a Service Bus relay.",
-        "description": "Creates a new Hybrid Connection using a Service Bus relay.",
-        "operationId": "WebApps_UpdateHybridConnection",
+        "summary": "Get the host configuration (host.json).",
+        "description": "Get the host configuration (host.json).",
+        "operationId": "WebApps_GetHostConfig",
         "parameters": [
           {
             "$ref": "#/parameters/resourceGroupNameParameter"
           },
           {
             "name": "name",
             "in": "path",
-            "description": "The name of the web app.",
+            "description": "Site name.",
             "required": true,
             "type": "string"
           },
           {
-            "name": "namespaceName",
-            "in": "path",
-            "description": "The namespace for this hybrid connection.",
-            "required": true,
-            "type": "string"
+            "$ref": "#/parameters/subscriptionIdParameter"
           },
           {
-            "name": "relayName",
-            "in": "path",
-            "description": "The relay name for this hybrid connection.",
-            "required": true,
-            "type": "string"
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Host configuration returned.",
+            "schema": {
+              "type": "object"
```

Comment:
This needs a definition. These `/properties/*` apis look to be anti-pattern, but it would be good to discuss with arm api review.

Reply:
The API returns the raw json file contents of the Functions hosting config (host.json) which lives in the root of a Function App site. So this is unstructured data.